src/schemas/components/SchemaFormPropertyString.vue

```diff
@@ -60,13 +60,13 @@
         'onUpdate:modelValue': update,
       })
     }
-    
+
     if (stringEnum) {
       return withProps(PCombobox, {
         modelValue: props.value,
         state: props.state,
         options: stringEnum.filter(isString),
-        'onUpdate:modelValue': (value) => asType(value, String),
+        'onUpdate:modelValue': (value) => update(asType(value, String)),
```

Comment:
Enums can have types that aren't strings - is this still correct?

Reply:
hmm this is specifically for properties of type `string`. So I think this is correct? But that probably means I need to account for enums for other things. Numbers and integers probably? What else could an enum be? A boolean?

Reply:
Add added support for integer enums in https://github.com/PrefectHQ/prefect-ui-library/pull/2114/commits/3eeca3f158e701b1607211a14f68743d3269bf05

Reply:
Hm ok I think we might need to expand support there but this is a strict improvement so going to approve

Reply:
Love to work with you on that if you have some time. Let me know!